### ISSUE TITLE
sysfs write failed /sys/devices/virtual/powercap/intel-rapl/intel-rapl:0/enabled

### ISSUE THREAD

**Issue by FadeMind** [intel/thermal_daemon#148]
I found issue https://github.com/intel/thermal_daemon/issues/113 but I have another PATH error:
```
sysfs write failed /sys/devices/virtual/powercap/intel-rapl/intel-rapl:0/enabled
```

```
grep -R . /sys/devices/virtual/powercap/intel-rapl/intel-rapl:0/enabled
0
```
```
[fademind@manjaro ~]$ systemctl status thermald.service -l
* thermald.service - Thermal Daemon Service
   Loaded: loaded (/usr/lib/systemd/system/thermald.service; enabled; vendor preset: disabled)
   Active: active (running) since Wed 2018-02-28 07:16:58 CET; 11h ago
 Main PID: 668 (thermald)
    Tasks: 2 (limit: 4915)
   CGroup: /system.slice/thermald.service
           `-668 /usr/bin/thermald --no-daemon --dbus-enable

lut 28 07:16:58 manjaro systemd[1]: Starting Thermal Daemon Service...
lut 28 07:16:58 manjaro thermald[668]: 13 CPUID levels; family:model:stepping 0x6:3a:9 (6:58:9)
lut 28 07:16:58 manjaro thermald[668]: Polling mode is enabled: 4
lut 28 07:16:58 manjaro systemd[1]: Started Thermal Daemon Service.
lut 28 07:16:58 manjaro thermald[668]: sysfs write failed /sys/devices/virtual/powercap/intel-rapl/intel-rapl:0/enabled
[fademind@manjaro ~]$ 
```

Thermald Stable 1.7.1-1 (official arch release)
Distro Manjaro Linux x64
CPU: Intel Core i7-3610QM (-MT-MCP-)

@spandruvada  can I just IGNORE this error, or it is something wrong with?

**Comment by emilio-aburto** [comment]
Hello, 
I found this same error along with one that seems fixed in #113 if I understood that correctly (probably not). Well, the interesting lines are:
```
may 06 04:55:07 speedy systemd[1]: Starting Thermal Daemon Service...
may 06 04:55:07 speedy systemd[1]: Started Thermal Daemon Service.
may 06 04:55:07 speedy thermald[1459]: 13 CPUID levels; family:model:stepping 0x6:45:1 (6:69:1)
may 06 04:55:07 speedy thermald[1459]: Polling mode is enabled: 4
may 06 04:55:07 speedy thermald[1459]: sysfs write failed /sys/devices/virtual/powercap/intel-rapl/intel-rapl:0/enabled
may 06 04:55:07 speedy thermald[1459]: sysfs read failed constraint_0_max_power_uw
```
Thermald 1.7.1-1
Distro Antergos
CPU: Intel Core i5-4200U

Is there more debugging info that I can provide? Just let me know :)

**Comment by spandruvada** [comment]
Since RAPL locked, the other controls in the /etc/thermald/thermal-cpu-cdev-order.xml will be able to take it over. You can follow suggestion of Hermanio and run some stress workload and check if there is any issue.
